src/components/Section.ts

```diff
@@ -32,6 +32,14 @@ export default (editor: grapesjs.Editor, { coreMjmlModel, coreMjmlView }: any) =
           'border-radius', 'border-top-left-radius', 'border-top-right-radius', 'border-bottom-left-radius', 'border-bottom-right-radius',
           'border', 'border-width', 'border-style', 'border-color'
         ],
+        traits:[
+          'id',
+          'title',
+          {
+            type: 'full-width',
+            label: 'Full width',
+          }
```

Comment:
Instead of creating your own trait, use the builtin checkbox?

```js
{
   type: 'checkbox',
   label: 'Full width',
   name: 'full-width',
   valueTrue: 'full-width',
   valueFalse: '',
}
```

Reply:
Right. Didn't know this was possible as it ins't in grapes documentation. Already did the change and commited